### ISSUE TITLE
Create a sparse matrix (interactions matrix) of user -> movie rating from the Netflix data set

### ISSUE THREAD

**Issue by audiodude** [noisebridge/MediaBridge#6]
LightFM requires a matrix, where the rows are the users and the columns are the movies. A '1' in a cell represents that the user liked that movie.

**Comment by audiodude** [comment]
We should consider whether we can or need to serialize this matrix, or if we can just recreate it each time.

**Comment by audiodude** [comment]
Basic algorithm:

- Create a numpy matrix with the right number of rows (users) and columns (movies)
- For each file (mv_00000X.txt):
  - read the movie id
  - For each line of the file:
    - Remap the user id
    - Update the matrix (skip any movie ratings that aren't 4 or 5)

Finally:
save the matrix to disk (pickle)

Some sample code:
```
n = 0
if id_ not in remap:
  remap[id_] = n
  n += 1

data[remap[id_]][movie_id] = 5
```

**Comment by audiodude** [comment]
As discussed a couple of weeks ago, the interactions matrix needs to be a `coo_matrix`: https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.coo_matrix.html

**Comment by audiodude** [comment]
Fixed by #10
